modules/cco-ccoctl-configuring.adoc

```diff
@@ -100,7 +100,7 @@ $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}'
 +
 [source,terminal]
 ----
-$ CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator' $RELEASE_IMAGE)
+$ CCO_IMAGE=$(oc adm release info -image-for='cloud-credential-operator' $RELEASE_IMAGE)
```

Comment:
@skrthomas I don't think this is the right fix. The issue mentions that the line has `--quay...` and it shouldn't have the `--` in it. It should just be `quay...`.

This is the section here: https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-upgrading_sts-mode-upgrading

In searching the repo, I see two CCO files that have a couple of the `--quay` instances that I think need to be changed:
* cco-ccoctl-creating-at-once.adoc
* cco-ccoctl-upgrading.adoc

So I would check those files, I think I see 3 total instances needing fixed.

Reply:
Yikes, I got this all backwards 🥴 Thanks for setting me str8. I'll get this fixed.

Reply:
@bergerhoffer Ok I think I got it right this time. Can you take another peep?

Reply:
Yep this looks better to me!

@nmasse-itix Does this PR look right to you?

Reply:
LGTM ! :+1: